CONTRIBUTING.md

```diff
@@ -51,7 +51,7 @@ Before you decide to create a new issue, please try the following:
 
 ### Please use our Bug Report Template
 
-In order to make the process easier, we've included a [sample bug report template](https://github.com/sendgrid/open-source-library-data-collector/.github/ISSUE_TEMPLATE) (borrowed from [Ghost](https://github.com/TryGhost/Ghost/)). The template uses [GitHub flavored markdown](https://help.github.com/articles/github-flavored-markdown/) for formatting.
+In order to make the process easier, we've included a [sample bug report template](https://github.com/sendgrid/open-source-library-data-collector/.github/ISSUE_TEMPLATE.md) (borrowed from [Ghost](https://github.com/TryGhost/Ghost/)). The template uses [GitHub flavored markdown](https://help.github.com/articles/github-flavored-markdown/) for formatting.
```

Comment:
I believe once published, the path will be: 
`https://github.com/sendgrid/open-source-library-data-collector/blob/master/ISSUE_TEMPLATE.md`
following the pattern from CONTRIBUTING here: `https://github.com/sendgrid/open-source-library-data-collector/blob/master/CONTRIBUTING.md`

Can you please change this?

Reply:
@mbernier : ah, I see what you mean! I was missing the `/blob/master` part of the path. I went and simplified the url to become a relative path so that the link works independently of what branch you are on, and just checked the link my develop branch to confirm that it works.

Would you strongly prefer the full url? I decided to try the relative URL method since it would do the same job in fewer characters.